app/javascript/components/forms/SurveyForm.vue

```diff
@@ -182,26 +182,55 @@ export default {
   },
```

Comment:
A lot of the edits here will be reorganisation and renaming - can talk through the meaningful changes if you like